SubPub/Tests/SubPubTests/TestManager.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  TestPublisher.swift
+//  
+//
+//  Created by Chad on 8/27/20.
+//
+
+import Foundation
+import SubPub
+
+public struct TestData: Equatable {
+    var id: String
+}
+
+public enum TestError: Swift.Error, Equatable {
+    case serviceError
+}
+
+public final class TestManager: NSObject {
```

Comment:
Does this need to be an NSObject?

Reply:
Hmmm. Maybe not. I was trying to be extra sure my deallocation test was valid, but I think it checks out as just a swift class object. Wish I could make that test more explicit, instead of just relying on maybe causing an error if something is wrong.

Reply:
actually, that's more about the subscriber object; maybe was a different reason for this one...